.go-version

```diff
@@ -1 +1 @@
-1.24.0
+1.24.1
```

Comment:
you can just touch this file to test go 1.24.1, this controls the version used in the makefile and other scripts, it will dynamically download go if needed

build/dependencies.yaml should have a group of related locations that should be updated together